docs/api.md

```diff
@@ -1273,8 +1273,8 @@ or updates existing device details for this user/session
 If no device `id` is specified,
 both `name` and `type` must be provided.
 If a device `id` is specified,
-at least one of `name`, `type`, `pushCallback` and `pushPublicKey`
-must be present.
+at least one of `name`, `type`, `pushCallback` or the couple `pushPublicKey` and `pushAuthKey`
+must be present. Be careful, `pushPublicKey` and `pushAuthKey` can change if `pushCallback` changes
```

Comment:
Reading that again, let me clarify:
It's safest to consider all three "joined". Chances are, these values will probably not change much, if at all. However, if they do, (e.g. via a re-registration event or a server side 4*0) it's best to replace all three with whatever values are returned by the client. Does that answer the question?